numpyro/examples/datasets.py

```diff
@@ -36,6 +37,11 @@
 ])
 
 
+DIPPER_VOLE = dset('dipper_vole', [
+    'https://d2hg8soec8ck9v.cloudfront.net/datasets/dipper_vole.zip',
```

Comment:
@neerajprad Could you help me upload this file [dipper_vole.zip](https://github.com/pyro-ppl/numpyro/files/5684306/dipper_vole.zip) to cloudfront?

Reply:
cc @neerajprad in case you missed to notification :D thanks!

Reply:
We don't have access to cloudfront anymore, unfortunately. :( We should discuss and figure out where to house these datasets (we only have read access to the remaining ones).

Reply:
@fehiepsi - I don't have an opinion on where we should put the dataset. My only concern is that it should have high availability for CI. We can start with whatever is convenient and if that works fine for some time, we can migrate the remaining datasets. Let me know if you want me to put this on my personal aws S3.

Reply:
Another easy option is raw.githubusercontent.com. We could move all datasets to https://github.com/pyro-ppl/datasets or something.

Reply:
I created the pyro-ppl/datasets repo. @fehiepsi - I think you were also in favor of this. Do you want to commit your zip file or upload the three files separately? I think they are small enough, so it shouldn't matter either ways.

Reply:
Thanks, @neerajprad! I'll upload the zip file there and update the link here.